StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp12/DocumentationRules/SA1611CSharp12UnitTests.cs

```diff
@@ -3,9 +3,61 @@
 
 namespace StyleCop.Analyzers.Test.CSharp12.DocumentationRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Testing;
     using StyleCop.Analyzers.Test.CSharp11.DocumentationRules;
+    using Xunit;
+    using static StyleCop.Analyzers.Test.Verifiers.CustomDiagnosticVerifier<StyleCop.Analyzers.DocumentationRules.SA1611ElementParametersMustBeDocumented>;
 
     public partial class SA1611CSharp12UnitTests : SA1611CSharp11UnitTests
     {
+        [Fact]
+        [WorkItem(3770, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3770")]
+        public async Task TestPrimaryClassConstructorMissingParametersAsync()
+        {
+            var testCode = @"
+/// <summary>
+/// Class.
+/// </summary>
+public class C(int param1, string param2) { }";
+
+            DiagnosticResult[] expectedResults = new[]
+            {
+                Diagnostic().WithLocation(5, 20).WithArguments("param1"),
+                Diagnostic().WithLocation(5, 35).WithArguments("param2"),
```

Comment:
These should use the markup syntax for location:

```suggestion
                Diagnostic().WithLocation(0).WithArguments("param1"),
                Diagnostic().WithLocation(1).WithArguments("param2"),
```

Reply:
I've changed all of my tests to use the markup syntax for location -- thanks for the explanation as I'm not intricately familiar with analyzer testing 🙂